example/components/config.js

```diff
@@ -1,16 +1,19 @@
-module.exports = function() {
-    return {
-        start: function start(cb) {
-            console.log('Starting config')
-            cb(null, {
-                service: {
-                    foo: 1
-                }
-            })
-        },
-        stop: function stop(cb) {
-            console.log('Stopping config')
-            cb()
+module.exports = options => {
+
+    const start = async () => {
+        console.log('Starting config')
+        return {
+            service: {
+                foo: 1
+            }
         }
     }
+
+    const stop = async () => {
```

Comment:
is it worth keeping this? systemic won't shout if there is no stop method (I seem to recall it's optional)